test/api_integration/config.js

```diff
@@ -40,6 +41,8 @@ export default async function ({ readConfigFile }) {
         '--optimize.enabled=false',
         '--elasticsearch.healthCheck.delay=3600000',
         '--server.xsrf.disableProtection=true',
+        `--plugin-path=${path.join(__dirname, 'fixtures', 'plugins', 'newsfeed')}`,
+        `--server.xsrf.whitelist=${JSON.stringify([`/api/_newsfeed-FTS-external-service-simulators/kibana`])}`,
```

Comment:
This looks like it will also need the `newsfeed.service` settings added, so the test server uses the fixtures served here as data to load in the UI

Reply:
Done.
Took a few hours to figure out why the CORS call kept failing... FTS swallows errors thrown by Hapi it seems.... but it now works.